### PR TITLE
RADFISH-77: Table to form route handler

### PR DESCRIPTION
**Pull request by gambinish**
Clicking on row should redirect to pre-populated form. This will either render out data from an endpoint (MSW) or from the localStorage cache.

This "should just work" when we get offline service worker issue resolved.

https://github.com/NMFS-RADFish/boilerplate/assets/35090461/676fd135-4c38-498a-ac0b-4eb5b765e932

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# IndexedDB
```
Comment (brockinit) [inline .env:1]: IndexDB **

Comment (brockinit) [reply .env:1]: I'm assuming you don't want to commit this file, right?

File: src/mocks/handlers.js [hunk 2]
```diff
@@ -83,4 +87,25 @@ export const handlers = [
       { status: 200 },
     );
   }),
+
+  http.get("/form/:id", ({ params }) => {
```
Comment (brockinit) [inline src/mocks/handlers.js:91]: Comments documenting this plz

**Review by brockinit — COMMENTED** [review]
Looks good. Obviously we need to get things hooked up to the local cache in order for this to work end-to-end

File: src/mocks/handlers.js (hunk 2 above)
Comment (gambinish) [reply src/mocks/handlers.js:93]: ✅

**Review by brockinit — APPROVED** [review]

**Pull request merged**